### DISCUSSION TITLE
Is it possible to increase the bit rate?

### DISCUSSION THREAD

**Discussion by 410011137** [neonbjb/tortoise-tts#128] [General]
Thank you for sharing this mind-blowing TTS! 
Would love to know what it takes to increase the output bit rate, to have a higher quality output?
I've read that this training model was trained mainly on audio books, as we probably know it has not so good audio quality (lower bitrate in general). I was wondering is it the bottleneck?

**Comment by neonbjb** [comment]
If I was going to attempt to improve audio quality, I would focus on improving the vocoder and the diffusion model, in that order. I would be curious to know how well Univnet would perform upsampling to 44kHz from the current MEL spec. I bet it would work pretty well.

**Reply by 410011137** [reply]
Thank you for the reply! Seem like I got a lot to read through.
